java/src/main/native/CMakeLists.txt

```diff
@@ -54,7 +54,9 @@ message(VERBOSE "CUDF_JNI: Statically link the CUDA runtime: ${CUDA_STATIC_RUNTI
 message(VERBOSE "CUDF_JNI: Build with GPUDirect Storage support: ${USE_GDS}")
 
 set(CUDF_SOURCE_DIR "${PROJECT_SOURCE_DIR}/../../../../cpp")
-set(CUDF_CPP_BUILD_DIR "${CUDF_SOURCE_DIR}/build")
+if (NOT CUDF_CPP_BUILD_DIR) 
```

Comment:
Was this supposed to be `if (NOT DEFINED CUDF_CPP_BUILD_DIR)`?  Otherwise there's no variable expansion in what is written that I can see.

Reply:
Oh nevermind, this should be OK as-is.